addons/bitmap-copy/addon.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Bitmap images copying",
+  "name": "Copy bitmap images",
```

Comment:
Oh yeah, this one.

The original name always felt awkward, but it’s not ideal for it to be a phrase either.

Reply:
The new one is definitely clear, IMO.

Reply:
Another option could be to keep the old one but remove the S from images.

Reply:
> Another option could be to keep the old one but remove the S from images.

I agree; to me that seems to be what is making the new one clearer.